packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -39,16 +40,16 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         uint32 lastVotingRound;
         // Denotes whether this is a governance request or not.
         bool isGovernance;
-        // The pendingRequestIndex in the pendingPriceRequests that references this PriceRequest. A value of UINT64_MAX
-        // means that this PriceRequest is resolved and has been cleaned up from pendingPriceRequests.
+        // The pendingRequestIndex in the pendingPriceRequestsIds that references this PriceRequest. A value of
+        // UINT64_MAX means that this PriceRequest is resolved and has been cleaned up from pendingPriceRequestsIds.
         uint64 pendingRequestIndex;
-        // Each request has a unique requestIndex number that is used to order all requests. This is the index within
-        // the priceRequestIds array and is incremented on each request.
-        uint64 priceRequestIndex;
-        // Timestamp that should be used when evaluating the request.
-        // Note: this is a uint64 to allow better variable packing while still leaving more than ample room for
-        // timestamps to stretch far into the future.
+        // The resolvedRequestIndex in the resolvedPriceRequestIds that references this PriceRequest.
+        uint64 resolvedRequestIndex;
```

Comment:
How does this differ from lastVotingRound? Would they always be the same?

Reply:
yup! I think this is redundant now. I have removed it.